### DISCUSSION TITLE
Authentication with API key / value

### DISCUSSION THREAD

**Discussion by bardossi** [pactumjs/pactum#312] [Q&A] [answered]
Hello, I was going through on the documentation but I couldn't find how to authenticate with an api-key, api-secret key-value pair. Probably it is something obvious what I did not find. Could someone help me out? Thanks.

**Comment by ASaiAnudeep** [comment, marked as answer]
Please use [withHeaders](https://pactumjs.github.io/api/requests/withHeaders.html) to pass custom headers to include api-key or  api-secret key-value pair.

**Reply by bardossi** [reply]
Thank you
